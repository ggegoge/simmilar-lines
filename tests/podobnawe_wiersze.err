ERROR 36
ERROR 37
ERROR 38
ERROR 39
ERROR 83
