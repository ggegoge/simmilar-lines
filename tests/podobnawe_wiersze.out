1
2
3
4 8 28 30 35 40 45 58 66 76 80 84 88
5
6
7
9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27
34
41
42
43
44
46 47
48
49 50 51
52 53
54 55
56 57
59
60 61 62 63 64 65
68 69 70 71 72 73 74 75
78
79
81
82
86 87
89
